ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/RecoveryMonitorTest2.java

```diff
@@ -27,8 +27,8 @@
 import com.arjuna.ats.arjuna.tools.RecoveryMonitor;
 import com.arjuna.ats.internal.jta.recovery.arjunacore.XARecoveryModule;
 import com.arjuna.ats.internal.jta.transaction.arjunacore.TransactionImple;
-import com.arjuna.ats.jta.logging.RecoveryRequired;
 import com.arjuna.ats.jta.logging.jtaLogger;
+import com.arjuna.ats.jta.recovery.RecoveryRequired;
```

Comment:
Where is this used?

Reply:
line no. 126

Reply:
[Line 126](https://github.com/jbosstm/narayana/pull/1600/files#diff-ee66272224729da4319abd39b5cbd2fbR126) reads as follows:

```java
            assertFalse(!xaRecoveryModule.isPeriodicWorkSuccessful());
```

Reply:
Your latest push has resolved this so I will mark it as resolved.

Reply:
Now I have deleted `RecoveryRequired` class and introduced new class variable, so line 126 is using some other class xaRecoveryModule to access recovery status.